training/culture-and-process/70_20_10_TIME.md

```diff
@@ -0,0 +1,44 @@
+70% 20% 10% Time in DP
+===========================
+
+As part of working in Digital Publishing, the development team share sprint time between feature work (70%) driven by product managers,
+technical workflows and tools (20%) driven by Technical Leads and development teams and self training (10%) driven by a developer.
```

Comment:
'personal development time' is a more commonly used phrase rather than self training. might help people put this in context with other corporate policies more easily :)

Reply:
Not sure its 100% grammar best practice, but i think an extra comma (or more punctuation somewhere) in this list might help because there are so many ands.

`development teams, and ...`

Reply:
I've created a bullet point list instead